qa/bwc/build.gradle

```diff
@@ -0,0 +1,181 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+
+apply plugin: 'opensearch.testclusters'
+apply plugin: 'opensearch.build'
+apply plugin: 'opensearch.rest-test'
+
+// Disable a few tasks that come with build
+build.enabled = false
+integTest.enabled = false
+test.enabled = false
+assemble.enabled = false
+dependenciesInfo.enabled = false
+
+dependencies {
+    api "org.opensearch:opensearch:${opensearch_version}"
+    compileOnly "org.opensearch.plugin:opensearch-scripting-painless-spi:${versions.opensearch}"
+    api group: 'commons-lang', name: 'commons-lang', version: '2.6'
+
+    api "org.apache.logging.log4j:log4j-api:${versions.log4j}"
+    api "org.apache.logging.log4j:log4j-core:${versions.log4j}"
+
+    testImplementation "org.opensearch.test:framework:${opensearch_version}"
+    testImplementation(testFixtures(rootProject))
+}
+
+
+def tmp_dir = project.file('build/private/artifact_tmp').absoluteFile
+tmp_dir.mkdirs()
+String knn_bwc_version = System.getProperty("bwc.version")
+
+// Task to pull k-NN plugin from archive
+task pullBwcPlugin {
```

Comment:
I feel like this section is unix os specific, can be a bummer in case we want to support something like win platform. did you try to find gradle native equivalent plugins/functions?

Reply:
sure, for us it works and isn't a blocker. it's more for customers on win or other non-unix like platforms. If it something that doesn't require a lot of efforts I think we need to do to that direction.

Reply:
Updated for non-unix